src/app/components/Footer/index.tsx

```diff
@@ -85,23 +103,46 @@ export const Footer = memo(() => {
                 <NoReleaseLink />
               ),
               CommitLink: (
-                <Anchor
-                  href={`${githubLink}commit/${process.env.REACT_APP_BUILD_SHA}`}
-                  label={process.env.REACT_APP_BUILD_SHA.substring(0, 7)}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                />
+                <span>
+                  <Anchor
```

Comment:
Accessibility wise, this element should be a button (Grommet Button)

Reply:
Good point, I have updated it. Not sure 100% if the props are correct though.